src/renderer/coremods/rdtComponentSourceFix/index.ts

```diff
@@ -0,0 +1,25 @@
+import { AnyFunction } from "src/types";
+
+function getOriginal(fn?: AnyFunction): AnyFunction | undefined {
+  return Function.toString.apply(fn).includes("objInjections.injections.get(funcName)")
```

Comment:
This function expects `fn` to be always of type 'function'. Sometimes this is not the case, for example when starting up the coremod the first time, `fn` will be undefined. Adding a check and returning `fn` if the typeof of `fn` is not a function is enough.